indico/modules/events/registration/models/forms.py

```diff
@@ -11,6 +11,7 @@
 from flask import session
 from sqlalchemy import orm, select
 from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.dialects.postgresql import UUID as SQL_UUID
```

Comment:
I'd `import uuid` instead of `from uuid import UUID` above and keep this one here un-aliased. The advantage of that is that all uuid columns use `UUID` (easier to search for in the codebase).